client.go

```diff
@@ -141,7 +141,7 @@ func (c *client) listen() {
 }
 
 func (c *client) handlePacket(remoteAddr net.Addr, packet []byte) error {
-	if protocol.ByteCount(len(packet)) > protocol.MaxPacketSize {
+	if protocol.ByteCount(len(packet)) > protocol.MaxReceivePacketSize {
```

Comment:
It cannot, and I already opened a new issue for that: #468. Afaik there's no signal, so we'll have to allocate 1 more byte in the packet buffer if we want to perform this check.